addons/binding/org.openhab.binding.max/ESH-INF/thing/bridge.xml

```diff
@@ -69,7 +101,7 @@
 		<label>Free Memory Slots</label>
 		<description>Free memory slots to store commands send to the devices
 		</description>
-		<category>FreeMemory</category>
+		<category>Humidity</category>
```

Comment:
No no no... Don't just pick any category because you like the icon! If there is no matching category yet, let's define if it makes sense to add a new one or live without a category for now.